vck/src/iosMain/kotlin/at/asitplus/wallet/lib/agent/DefaultCryptoService.kt

```diff
@@ -1,58 +0,0 @@
-@file:OptIn(ExperimentalForeignApi::class, ExperimentalNativeApi::class)
-
-package at.asitplus.wallet.lib.agent
-
-import at.asitplus.KmmResult
-import at.asitplus.signum.indispensable.io.Base64Strict
-import at.asitplus.signum.indispensable.josef.JsonWebKey
-import at.asitplus.signum.indispensable.josef.JweAlgorithm
-import at.asitplus.signum.indispensable.josef.JweEncryption
-import io.matthewnelson.encoding.core.Encoder.Companion.encodeToString
-import kotlinx.cinterop.ExperimentalForeignApi
-import kotlin.experimental.ExperimentalNativeApi
-
-
-/**
- * Default implementation of a crypto service for iOS.
- *
- * The primary goal is to provide a minimal implementation so that unit tests in the `commonTest` module run successfully.
- *
- * Beware: It does **not** implement encryption, decryption, key agreement and message digest correctly.
- */
-@Suppress("UNCHECKED_CAST")
-actual open class PlatformCryptoShim {
```

Comment:
Won't this lead to compilation errors on clients upgrading from vc-k 5.5.0 to 5.6.0? Can we leave this class and deprecate it?

Reply:
you'll need to update anyway, because of signum breaking changes